src/components/SiteFooter/SiteFooter.stories.tsx

```diff
@@ -1,11 +1,41 @@
-import React from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
+import { LinkProps } from '../Link/Link';
 
 // Import React component.
 import SiteFooter from './SiteFooter';
 
-// Menu Example Data.
-import { PrimaryMenuData, SecondaryMenuData } from './DataExample';
+// This data is used as an example.
+const PrimaryMenuData: LinkProps[] = [
+  {
+    children: 'Primary Link 1',
+    href: 'https://gov.wales',
+    ['aria-label']: 'Test aria label',
+    target: '_blank',
```

Comment:
Remove target: '_blank'.  Primary Menu items should not link to an external site or new window.
Secondary menu items can open new window, so it okay to be be placed on one of these links as an example.